### PR TITLE
Add tests for the redshift slider

### PR DESCRIPTION
**Pull request by javerbukh**
Add some tests for a particular part of the redshift slider.

### CODE REVIEW

File: jdaviz/configs/specviz/plugins/redshift_slider/tests/test_redshift_slider.py
Comment (pllim) [inline jdaviz/configs/specviz/plugins/redshift_slider/tests/test_redshift_slider.py:21]: Same as my other review comment before: Avoid use of random or set a seed.

**Review by pllim — COMMENTED** [review]
I see a lot of code repetition. Perhaps you want to consider https://docs.pytest.org/en/6.2.x/parametrize.html

File: jdaviz/configs/specviz/plugins/redshift_slider/tests/test_redshift_slider.py
Comment (javerbukh) [reply jdaviz/configs/specviz/plugins/redshift_slider/tests/test_redshift_slider.py:21]: Got it, I addressed this in my other PR. I will rebase once that is merged.

**Comment by javerbukh** [comment]
This PR needs to be rebased once #626 is merged.

**Review by rosteen — APPROVED** [review]
I always appreciate new tests! Look good to me.

**Comment by rosteen** [comment]
@pllim Your comments were addressed here, right? Want to give it your stamp of approval?

**Review by pllim — APPROVED** [review]
Looks like my advise to https://docs.pytest.org/en/6.2.x/parametrize.html was not taken, but it is a small enough PR that it doesn't matter much to me. Just increase tech debt in the future if you want to add more similar tests. I guess you can refactor then when needed.

**Comment by pllim** [comment]
p.s. Not sure why RTD didn't fire off but it shouldn't matter. Does need admin overwrite but admin is on vacation. So force push might work faster.

**Review by ibusko — APPROVED** [review]
I agree that repeated code could be refactored. But I also think that in the way it is now, it's easier to read.

**Comment by pllim** [comment]
I am going to close/open to kick off RTD again. Merge is blocked otherwise.

**Pull request merged**
